utils/statistics-flow-rejection-reason-results.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var db = require('../db');
+
+module.exports = function (reasons) {
+	return reasons.map(function (reason) {
+		var result = [], reasonsConcat = [];
+		reason.rejectionReasons.forEach(function (reasonItem) {
+			reasonItem.types.forEach(function (type) {
+				reasonsConcat.push(type);
+				if (type === 'other' && reasonItem.value) {
+					reasonsConcat.push(reasonItem.value);
+				}
+			});
+		});
+		result.push(reasonsConcat);
+		result.push(reason.hasOnlySystemicReasons ? '*' : '');
+		result.push('');
```

Comment:
I don't expect much data here, but let's be prepared just in case :)... so basically I think we should show sth here when it is a number greater than 1.